Add repository links to the README.rst file (#241)
google/skywater-pdk: README.rst

See both the `Known Issues <docs/known_issues.rst>`_ section and the `SkyWater PDK GitHub issue list <https://github.com/google/skywater-pdk/issues>`_ to get more detailed information around currently known issues.

Resources
=========

> The latest SkyWater SKY130 PDK design resources can be downloaded from the following repositories:

Reviewer: ```suggestion
The latest SkyWater SKY130 PDK design resources can be viewed at the following locations:
```
